.ruby-version

```diff
@@ -1 +0,0 @@
-3.1.4
```

Comment:
Did you mean to delete this file?

Reply:
oh. I have to install a new version of Ruby and I thought this file was create due to installation. I reverted this.